src/types/camera/CameraPreset.php

```diff
@@ -49,8 +53,14 @@ public function getPitch() : ?float{ return $this->pitch; }
 
 	public function getYaw() : ?float{ return $this->yaw; }
 
+	public function getRotationSpeed() : ?float { return $this->rotationSpeed; }
+
+	public function getSnapToTarget() : ?bool { return $this->snapToTarget; }
```

Comment:
Usually boolean fields are prefixed with `is` or `has` (`hasSnapToTarget`)

Reply:
Yes, but this is not always boolean. It also can be nullable.

Reply:
Alright thanks